modynclient/client/internal/grpc_handler.py

```diff
@@ -32,7 +33,9 @@ def init_supervisor(self) -> None:
         )
 
         if not grpc_connection_established(self.supervisor_channel):
-            raise ConnectionError(f"Could not establish gRPC connection to supervisor at {supervisor_address}.")
+            raise ConnectionError(
+                f"Could not establish gRPC connection to supervisor at {supervisor_address}."
```

Comment:
Sorry, these changes come from my on-save linters that are not configured by a `pyproject.toml` here.